build/Makefile

```diff
@@ -23,9 +23,9 @@ SCX_INTERMEDIATE_DIR := $(SCX_DIR)/intermediate/$(BUILD_CONFIGURATION)
 DSC_TARGET_DIR := $(DSC_DIR)/release
 
 RUBY_DIR := $(BASE_DIR)/source/ext/ruby
+RUBY_VER := 2.3.0
```

Comment:
Added a comment about what this version number refers to, because it can be confusing, since this line says 2.3.0 when our tagged Ruby version has a different teeny number.